cds_ils/migrator/eitems/api.py

```diff
@@ -169,8 +169,10 @@ def process_files_from_legacy():
                 )
 
                 # get filename
-                file_name = file_dump["description"]
-                if not file_name:
+                if "description" in file_dump:
```

Comment:
Am I a bit worried that there is no sanitization at all of this before setting it as filename.
It is not very important because in Invenio it is just a string in a db table, but what if it is super long?
Do you have any example in CDS?